reasoner_validator/biolink/__init__.py

```diff
@@ -695,7 +695,8 @@ def validate_infores(self, context: str, edge_id: str, identifier: str):
                 identifier=identifier,
                 edge_id=edge_id
             )
-        # TODO: replace with another function to validate the infores in registered?
+        # TODO: infores is causing too much instability for now so support has been removed from BMT
+        # TODO: reimplement using YAML version of infores catalog
```

Comment:
Seems like a good way to do it if `bmt` does not provide a way to easily retrieve the infores catalog anymore

Reply:
indeed; this should happen in a separate PR, I think?   And we should decide if we want this kind of extra testing as it requires a dependency on Biolink Model.